### DISCUSSION TITLE
SMTP setup trouble for Gmail

### DISCUSSION THREAD

**Discussion by shiryu1031** [dani-garcia/vaultwarden#2181] [General]
I am having trouble setting up to be able to send email from my Gmail.
I want to use organization, but unable to send out emails.

I am getting this error when I try to send out a test email
![Capture](https://user-images.githubusercontent.com/95537569/147530885-6a4cbd95-724e-4270-8d81-545256e20a04.PNG)

This is my setting.
![Capture](https://user-images.githubusercontent.com/95537569/147530823-2b78fadf-a4b4-4abb-aa1a-4809859e5e53.PNG)

I greatly appreciate if someone can help me with this. Thank you very much!

**Comment by alexdelprete** [comment]
I have the same config, and it works fine.

The only problem I see in yours is that in From Name you have to use an email address, not only a label.
Use something like `Vaultwarden <name@domain.com>`

![image](https://user-images.githubusercontent.com/7027842/147747394-7adcfa7b-b62a-4612-9edb-ff8f29d6102d.png)

**Comment by shiryu1031** [comment]
nevermind, I just fixed it.
Thank you very much for your help.

all setting were correct, the name wasn't the problem.
I just didn't open port 465.
